taker/src/main.rs

```diff
@@ -126,10 +126,10 @@ struct Opts {
     #[clap(subcommand)]
     network: Option<Network>,
 
-    #[clap(short, long, parse(try_from_str = parse_umbrel_seed))]
-    umbrel_seed: Option<[u8; 32]>,
+    #[clap(short, long, parse(try_from_str = parse_app_seed))]
```

Comment:
📯
imho this asks for a changelog update

Reply:
thank you for reminder, it is indeed easier to update it as-we-go.